chat-plugins/helptickets.js

```diff
@@ -446,7 +446,9 @@ exports.pages = {
 exports.commands = {
 	requesthelp: 'helpticket',
 	helprequest: 'helpticket',
-	report: 'helpticket',
+	report: function (target, room, user, connection) {
+		return this.parse('/helpticket');
```

Comment:
this doesnt do what you want it to do; this just means that doing `/report` will do what it already does

Reply:
and seeing as the point of the pull request is to have `/report` open the Ticket window, this is what it already does, which makes this PR useless?

Reply:
i tested it on localhost and '/report hawkie' opened the view-help-request room like it should, currently the server just returns 'The command "/report hawkie" does not exist..' etc

Reply:
The ticket window currently doesn't open if you do `/report target`, which a lot of people do as they're using the command to report a user. With this change `/report` ignores the target and just opens the ticket window.

Reply:
ok but what about the rest of the aliases?

Reply:
the other aliases were left alone so `/ht ban/unban` etc are unaffected. `/report` is the only alias that causes users problems accessing the ticket window.  It does mean staff won't be able to use `/report [command]` though.